src/grdeviceside.cpp

```diff
@@ -323,6 +342,30 @@ namespace rhost {
                 return (*_active_plot).get();
             }
 
+            plot* plot_history::get_plot(const boost::uuids::uuid& plot_id) {
+                for (auto it = _plots.begin(); it != _plots.end(); ++it) {
```

Comment:
The thing is I need an iterator, because `_active_plot` that I want to set is an interator. 

`for (auto&& plot : _plots)` gets me a reference to a `unique_ptr`.